website/pages/style.css

```diff
@@ -82,6 +82,237 @@
+  }
+
+  pre,
+  table {
+    break-inside: avoid;
+  }
+
+  pre {
+    white-space: pre-wrap;
+  }
+}
+
+@media print {
+  /* @todo: remove alongside @hashicorp/react-global-styles/_temporary-to-remove/layout.css */
+  .g-container {
+    max-width: 34em;
+    word-break: break-word;
+  }
+
+  /* @todo: remove alongside @hashicorp/react-global-styles/_temporary-to-remove/tables.css */
+  table {
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+}
+
+/* @todo: move print styles to @hashicorp/react-global-styles/global.css */
+@media print {
+  pre code,
+  code,
+  pre {
+    font-weight: inherit;
+  }
+
+  pre {
+    background: transparent;
+    box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.15);
+    color: inherit;
+    padding: 0.5em;
+
+    & > code {
+      white-space: inherit;
+    }
+  }
+}
+
+/* @todo: move print styles to @hashicorp/react-content/dist/style.css */
+@media print {
+  .g-content {
+    & a {
+      color: inherit;
+      font-weight: 700;
+
+      &:not(.anchor) {
+        &::after {
+          background-color: transparent;
+          position: static;
+          opacity: 1;
+        }
+      }
+
+      &[href^='http'] {
+        &::after {
+          content: ' <' attr(href) '>';
+          font-size: 0.8em;
+          font-style: italic;
+          letter-spacing: -0.01875em;
+          vertical-align: top;
+        }
+      }
+
+      &:not([href^='http']) {
+        text-decoration: underline;
+      }
+
+      & > code {
+        color: inherit;
+        font-weight: 700;
+
+        &::before,
+        &::after {
+          content: none;
+        }
+      }
+    }
+
+    & h1,
+    & h2,
+    & h3,
+    & h4,
+    & h5,
+    & h6 {
+      & code {
+        background: none;
+        font-size: 1em;
+        padding: 0;
+      }
+    }
+
+    & h2 {
+      margin: 1em 0 0;
+    }
+
+    & h3 {
+      margin: 1em 0 0;
+      padding-bottom: 0.25em;
+    }
+
+    & img {
+      box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15);
+      margin: 1em 0 0;
+    }
+
+    & ol,
+    & ul {
+      margin: 1em 0 0;
+
+      & li {
+        margin-bottom: 0;
+        margin-top: 0.5em;
+
+        & p:first-child {
+          margin-top: 0;
+        }
+      }
+    }
+
+    & p {
+      margin: 1em 0 0;
+    }
+
+    & pre {
+      background-color: transparent;
+      border-radius: 0;
+      margin: 1.5em 0 0;
+
+      & code {
+        background: transparent;
+        color: inherit;
+      }
+    }
+
+    & dd,
+    & dt,
+    & li,
+    & p,
+    & td,
+    & th {
+      & > :not(pre) code,
+      & > code {
+        background: transparent;
+        font-weight: 700;
+        padding: 0;
+      }
+    }
+
+    & .alert.alert-danger,
+    & .alert.alert-info,
+    & .alert.alert-success,
+    & .alert.alert-warning {
+      background-color: transparent;
+    }
+  }
+}
+
+/* @todo: move print styles to @hashicorp/react-global-styles/code-highlighting.css */
+@media print {
+  .hljs {
+    & * {
+      color: inherit;
+    }
+  }
```

Comment:
This is a nit, but could this be:
```css
.hljs * {
  color: inherit;
}

Reply:
I like the way `&` makes combinator relationships visually consistent by pushing them to the beginning of the selector. And I like how the nesting allows for other rules to be added before or after it without affecting its own indentation (and therefore the perceived changes in that kind of commit).

Reply:
that's fair! happy to approve as is, good to know this is more a style thing than anything else 👍